app/views/create-route.html

```diff
@@ -27,6 +27,13 @@
                             services="services"
                             show-name-input="true">
                         </osc-routing>
+                        <label-editor
+                            labels="labels"
+                            expand="true"
+                            can-toggle="false"
+                            help-text="Labels for this route.">
+                        </label-editor>
+                        <a href="" ng-click="copyServiceLabels()">Copy service labels</a>
```

Comment:
Let's merge the other one first, and I'll rebase this one. About to push updates for #978